### PR TITLE
[CDAP-6236] CLI commands with space in path

### PR DESCRIPTION
**Pull request by liu-joe**

### CODE REVIEW

File: cdap-cli/src/main/java/co/cask/cdap/cli/util/FilePathResolver.java [hunk 1]
```diff
@@ -52,14 +52,18 @@ public FilePathResolver() {
...
    * @param path bash-style path
    * @return {@link File} of the resolved path
    */
   public File resolvePathToFile(String path) {
     path = resolveVariables(path);
 
+    if (path.contains("\\ ")) {
+      path = path.replace("\\ ", " ");
+    }
+
```
Comment (anew) [inline cdap-cli/src/main/java/co/cask/cdap/cli/util/FilePathResolver.java:66]: Does the file name completion work for files with spaces? And does it auto-complete to path\ with\ spaces?

Comment (liu-joe) [reply cdap-cli/src/main/java/co/cask/cdap/cli/util/FilePathResolver.java:66]: auto complete doesn't add the \ in front of spaces

Comment (anew) [reply cdap-cli/src/main/java/co/cask/cdap/cli/util/FilePathResolver.java:66]: what does it do? I did a quick try with a file named "a b", and the auto-complete only showed the "a" (up to the first blank).

Comment (anew) [reply cdap-cli/src/main/java/co/cask/cdap/cli/util/FilePathResolver.java:66]: but I guess the auto-complete should be compatible with what the CLI understands.

**Comment by anew** [comment]
Another question I have is whether the escaping is required. Can we just enclose the file name in double quotes? That is what we do for other things like runtime arguments: you can give a single argument as key=value, but if you have more than one, you need to give them as "k1=v1 k2=v2". Would quoting also work for file names?

**Comment by anew** [comment]
For example, I can do this: 
```
cdap (http://andreas@censored.host.name:11015/namespace:test)> load stream str "a b" text/plain
Successfully loaded file to stream 'str'
```

**Comment by anew** [comment]
Please see my comment in the Jira. Perhaps this is a won't fix/user error

**Comment by liu-joe** [comment]
Yeah, you're right, it works if you just quote the string. But i was playing around with the jline completer, and I can probably get the completer to work with spaces, so that's one advantage over quoting the path

**Pull request closed without merging**